packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -202,6 +202,28 @@ export default class TurnDiscovery {
    * @returns {Promise}
    */
   doTurnDiscovery(meeting, isReconnecting) {
+    const reachabilityData = window.localStorage.getItem(REACHABILITY.localStorage);
+    if (reachabilityData) {
+      try {
+        const reachabilityResult = JSON.parse(reachabilityData);
+
+        const reachable = Object.values(reachabilityResult).some((result: {udp: {reachable: 'true'| 'false'}, tcp: {reachable: 'true'| 'false'}}) => {
+          if (result.udp.reachable === 'true' || result.tcp.reachable === 'true') {
```

Comment:
I'm not sure if it's guaranteed that every result will have udp and tcp checks done, I think it depends on what Orpheus gives us, so it would be safer to do `result.udp?.reachable` and `result.tcp?.reachable` here